app/src/main/java/com/quarkworks/apartmentgroceries/profile/ProfileDetailActivity.java

```diff
@@ -43,21 +45,24 @@
 import bolts.Continuation;
 import bolts.Task;
 
-public class UserDetailActivity extends AppCompatActivity {
-    private static final String TAG = UserDetailActivity.class.getSimpleName();
+public class ProfileDetailActivity extends AppCompatActivity implements PopupDialog.NoticeDialogListener {
```

Comment:
Instead of implementing the listener, you can create a reference in code for it like you would for a View.OnClickListener.